test/config.json

```diff
@@ -23,9 +23,51 @@
           "remote": true
         }
       ]
+    },
+    {
+      "file": "test/specs/multipleBrowsersDocumentUpload.js",
+      "timeout": 120000,
+      "browsers": [
+        {
+          "browserName": "firefox",
+          "browser_version": "74.0",
+          "os": "OS X",
+          "os_version": "Catalina",
+          "resolution": "1920x1080",
+          "remote": true
+        }
+      ]
+    },
+    {
+      "file": "test/specs/multipleBrowsersDocumentUpload.js",
+      "timeout": 120000,
+      "browsers": [
+        {
+          "browserName": "firefox",
+          "browser_version": "74.0",
+          "os": "Windows",
+          "os_version": "10",
+          "resolution": "1920x1080",
+          "remote": true
+        }
+      ]
+    },
+    {
+      "file": "test/specs/multipleBrowsersDocumentUpload.js",
+      "timeout": 120000,
+      "browsers": [
+        {
+          "browserName": "edge",
+          "browser_version": "79.0 beta",
```

Comment:
Just noticed this, but why `79.0 beta` version for Edge?

Reply:
Actually, the latest version of Edge BS supports on Windows 8 is 81 beta so I changed it. They also have 80.0.

Reply:
I would suggest to go with the latest stable version (80.0) rather than the beta.

Reply:
Done :shipit: